data/protocols/250/Beethoven-x.json

```diff
@@ -17,7 +17,7 @@
     },
     "de": {
       "name": "Beethoven-x",
-      "description": "Beethoven X is an automated market maker protocol on the Fantom network. It's a decentralized liquidity aggregator where users can add their assets to several different liquidity pools and earn fees."
+      "description": "Beethoven X ist ein automatisiertes Market-Maker-Protokoll im Fantom-Netzwerk. Es ist ein dezentraler Liquiditätsaggregator, bei dem Nutzer ihre Vermögenswerte zu verschiedenen Liquiditätspools hinzufügen und Gebühren verdienen können"
```

Comment:
AMM is not capped, I think? Fantom network not Network. Liquidity aggregator as well, and so on. Only cap proper nouns correct?

Reply:
Also add a `.`

Reply:
In German, usually all names and nouns need to be capitalized. As "automated" is an adjective it is written in small letters, "market maker protocol" and "Fantom network" are all names or nouns that's why I capitalized them.